blocks/editable/index.js

```diff
@@ -387,6 +387,7 @@ export default class Editable extends Component {
 	}
 
 	getContent() {
+		return this.editor.getContent();
```

Comment:
@youknowriad No it's being worked at, I added this because redux states had react elements which I was not able to serialize. This works fine and output string but we are working on this for proper structure.